edb/schema/_types.py

```diff
@@ -4,28 +4,30 @@
 
 from __future__ import annotations
 
-import uuid
+from edb.common import uuidgen
 
 
 TYPE_IDS = {
-    'anytype': uuid.UUID('00000000-0000-0000-0000-000000000001'),
-    'anytuple': uuid.UUID('00000000-0000-0000-0000-000000000002'),
-    'std': uuid.UUID('00000000-0000-0000-0000-0000000000f0'),
-    'empty-tuple': uuid.UUID('00000000-0000-0000-0000-0000000000ff'),
-    'std::uuid': uuid.UUID('00000000-0000-0000-0000-000000000100'),
-    'std::str': uuid.UUID('00000000-0000-0000-0000-000000000101'),
-    'std::bytes': uuid.UUID('00000000-0000-0000-0000-000000000102'),
-    'std::int16': uuid.UUID('00000000-0000-0000-0000-000000000103'),
-    'std::int32': uuid.UUID('00000000-0000-0000-0000-000000000104'),
-    'std::int64': uuid.UUID('00000000-0000-0000-0000-000000000105'),
-    'std::float32': uuid.UUID('00000000-0000-0000-0000-000000000106'),
-    'std::float64': uuid.UUID('00000000-0000-0000-0000-000000000107'),
-    'std::decimal': uuid.UUID('00000000-0000-0000-0000-000000000108'),
-    'std::bool': uuid.UUID('00000000-0000-0000-0000-000000000109'),
-    'std::datetime': uuid.UUID('00000000-0000-0000-0000-00000000010a'),
-    'std::local_datetime': uuid.UUID('00000000-0000-0000-0000-00000000010b'),
-    'std::local_date': uuid.UUID('00000000-0000-0000-0000-00000000010c'),
-    'std::local_time': uuid.UUID('00000000-0000-0000-0000-00000000010d'),
-    'std::duration': uuid.UUID('00000000-0000-0000-0000-00000000010e'),
-    'std::json': uuid.UUID('00000000-0000-0000-0000-00000000010f'),
+    'anytype': uuidgen.UUID('00000000-0000-0000-0000-000000000001'),
+    'anytuple': uuidgen.UUID('00000000-0000-0000-0000-000000000002'),
+    'std': uuidgen.UUID('00000000-0000-0000-0000-0000000000f0'),
+    'empty-tuple': uuidgen.UUID('00000000-0000-0000-0000-0000000000ff'),
+    'std::uuid': uuidgen.UUID('00000000-0000-0000-0000-000000000100'),
+    'std::str': uuidgen.UUID('00000000-0000-0000-0000-000000000101'),
+    'std::bytes': uuidgen.UUID('00000000-0000-0000-0000-000000000102'),
+    'std::int16': uuidgen.UUID('00000000-0000-0000-0000-000000000103'),
+    'std::int32': uuidgen.UUID('00000000-0000-0000-0000-000000000104'),
+    'std::int64': uuidgen.UUID('00000000-0000-0000-0000-000000000105'),
+    'std::float32': uuidgen.UUID('00000000-0000-0000-0000-000000000106'),
+    'std::float64': uuidgen.UUID('00000000-0000-0000-0000-000000000107'),
+    'std::decimal': uuidgen.UUID('00000000-0000-0000-0000-000000000108'),
+    'std::bool': uuidgen.UUID('00000000-0000-0000-0000-000000000109'),
+    'std::datetime': uuidgen.UUID('00000000-0000-0000-0000-00000000010a'),
+    'std::local_datetime': uuidgen.UUID(
+        '00000000-0000-0000-0000-00000000010b'
+    ),
```

Comment:
You should actually update the `edb/tools/gen_types.py` command -- it's used to generate this file.

I'd suggest to update it to do this (to make sure we don't need to break lines to fit in 79):

```python
from edb.common import uuidgen

UUID = uuidgen.UUID

TYPE_IDS = {
    'anytype': UUID('00000000-0000-0000-0000-000000000001'),
    'anytype': UUID('00000000-0000-0000-0000-000000000001'),
   ...
```

Reply:
OK, I'll do that.